TotalCrossVM/android/app/build.gradle

```diff
@@ -14,8 +14,7 @@ apply plugin: 'com.android.application'
 def buildType // stores build type, debug or release
 
 android {
-    compileSdkVersion 28
-    ndkVersion "21.0.6113669"
```

Comment:
we should enforce the usage of a ndk version to make sure the results are the same, either keep 21.0.6113669 or move to the latest one

Reply:
The gradle plugin takes care of that